README.md

```diff
@@ -63,6 +59,10 @@ There are many ways to [deploy KEDA including Helm, Operator Hub and YAML files]
 
 Interested to learn more? Head over to [keda.sh](https://keda.sh).
 
+### Become a listed KEDA user!
```

Comment:
```suggestion
## Become a listed KEDA user!
```
This probably is not a subsection of Documentation, right?

Reply:
This was intentional as this is more documentation related but can make it more important if we want to.

Reply:
Then doc-toc is right here: https://github.com/kedacore/keda/pull/1112#discussion_r484867125

The `### Become a listed KEDA user!` will be subsection of `## Documentation`

Reply:
Yes I know, it was intentional :)

Reply:
I just want to say that assuming your intention (which I think is ok) the doctoc tool generated a proper table of content. If we do want to list only "main sections" we may consider changing `--maxlevel` to `1` in pre-commit config for doctoc

Reply:
IMHO `Become a listed KEDA user!` doesn't have to be a subsection of `## Documentation`

But I am ok with both options.

Reply:
Updated with a new proposal